src/Esprima/Ast/JSXAttribute.cs

```diff
@@ -0,0 +1,23 @@
+﻿using Esprima.Utils;
+
+namespace Esprima.Ast
+{
+    public sealed class JSXAttribute : JSXExpression
```

Comment:
I've change all of them except `Nodes` because ;
![CleanShot 2022-05-06 20 46 20](https://user-images.githubusercontent.com/8670548/167187089-97d13d3a-e1e8-4bd9-981e-609c0dcf566c.png)

Reply:
Sounds reasonable, AST tree JSON output should follow all-caps casing for JSX, like other parsers output it too